.github/workflows/desktop-build.yml

```diff
@@ -125,20 +125,17 @@ jobs:
     continue-on-error: ${{matrix.allow-failure == 'yes'}}
     env:
       NAME: ${{matrix.distro}}${{matrix.version}}
-      CACHE: /tmp/${{matrix.distro}}${{matrix.version}}-cache    # ${{runner.temp}} does not work?
+      CACHE: ${{github.workspace}}/.cache/${{matrix.distro}}${{matrix.version}}   # directory for caching docker image and ccache
       # Cache size over the entire repo is 10Gi:
       # https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows#usage-limits-and-eviction-policy
       CCACHE_SIZE: 500M
+      CCACHE_COMPILERCHECK: content
```

Comment:
IMO, it would be nice to add an inline comment about what is this for and why it is needed

Reply:
I did revert this change actually.

It seems to be rather an issue when using Android NDK and misleading suggestion in React Native docs. See also e.g. https://github.com/hendrikmuhs/ccache-action/issues/94#issuecomment-2286488872